### PR TITLE
Create Fact “wild-bergamot-bee-balm”

### PR DESCRIPTION
**Pull request by NicolasRobertson**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/facts/wild-bergamot-bee-balm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Wild Bergamot (bee balm)
```
Comment (floogulinc) [inline src/facts/wild-bergamot-bee-balm.md]: Can we just call this "Wild Bergamot" and say that its "often called bee balm" in the fact?

File: src/facts/wild-bergamot-bee-balm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+image:
+  src: ../images/wildbergamot.jpg
+  url: https://www.flickr.com/photos/tgpotterfield/28237843773
+  license: CC by-NC-SA 2.0
```
Comment (floogulinc) [inline src/facts/wild-bergamot-bee-balm.md]:
```suggestion
  license: CC BY-NC-SA 2.0
```

File: src/facts/wild-bergamot-bee-balm.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  alt: Wild Bergamot (bee balm)
+  creator: Tom Potterfield
+source:
+  name: "Bees: An Identification and Native Plant Forage Guide"
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+category: Plant
+---
+Bees often seek nectar from this flower, though it offers both nectar and pollen.
+
+Common bees attracted include bumble, digger, long-horned, leaf-cutter, and metallic green sweat bees.
```
Comment (floogulinc) [inline src/facts/wild-bergamot-bee-balm.md]:
```suggestion
Bees often seek nectar from this flower, though it offers both nectar and pollen. Common bees attracted include bumble, digger, long-horned, leaf-cutter, and metallic green sweat bees.
```

**Review by floogulinc — CHANGES_REQUESTED** [review]

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to just `wild-bergamot.md`

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to just `wild-bergamot.md`

**Review by floogulinc — APPROVED** [review]

**Review by kklamberty — APPROVED** [review]

**Pull request merged**